conda/_vendor/toolz/itertoolz.py

```diff
@@ -4,8 +4,9 @@
 import operator
 from functools import partial
 from random import Random
-from .compatibility import map, filterfalse, zip, zip_longest, iteritems, filter
-from .utils import no_default
+from conda._vendor.toolz.compatibility import (map, filterfalse, zip, zip_longest, iteritems,
+                                 filter)
+from conda._vendor.toolz.utils import no_default
```

Comment:
Same thing as the dicttoolz module above.

Reply:
vendoring magic